cmd/ctrlc/root/run/exec/exec.go

```
@@ -11,34 +16,76 @@ import (
 			if err != nil {
 				return fmt.Errorf("failed to create API client: %w", err)
 			}
+			if name == "" {
+				return fmt.Errorf("name is required")
+			}
+			if workspaceId == "" {
+				return fmt.Errorf("workspace is required")
+			}
+			interval := viper.GetString("interval")
+			if interval == "" {
+				interval = "10s"
```

golangci-lint (1.64.8)

ineffectual assignment to interval

(ineffassign)